packages/core/test/actor.test.ts

```diff
@@ -361,7 +367,8 @@ describe('communicating with spawned actors', () => {
     parentService.start();
   });
 
-  it('should be able to communicate with arbitrary actors if sessionId is known', (done) => {
+  // TODO: This is an invalid use-case; consider removing
```

Comment:
I'm looking at just the title of this test:
> should be able to communicate with arbitrary actors if sessionId is known

why this would be considered invalid? Could u give more context?

Reply:
We talked about an interpreted machine creating an implicit "system". If we create two machines completely separately, we should create two implicit systems, instead of magically relying on some global "system" (aka registry of machines).

I can refactor this test so that the other machine is created _from_ the first machine. With the `ActorRef` refactor from master merged into this one, this test breaks though.

I don't think this is a use-case worth supporting; better to be explicit that actors can only communicate if they come from the same system.